File: migration/workspace/workspace-wk2/commandLineArguments/argv2.c
/*
 * argv2.c
 *
 * This program iterates over strings and
 * characters within strings
 */
 
 #include <stdio.h>
 #include <string.h>
 #include <cs50.h>
 
 int main(int argc, string argv[])
 {
 
    // Iterates over strings in argv
    for (int i = 0; i < argc; i++)
    {
         // Iterates over characters of current string
        for (int j = 0, n = strlen(argv[i]); j < n; j++)
        {
             // Prints the j'th character in i'th string
            printf("%c\n", argv[i][j]);
        }
        
        printf("\n"); // Prints a space between each string
    }
    
    return 0;
 }

File: practice/01_old/practice-wk6/argv/argv2.py
# argv2.py
# Sat, 02.02.19 12:49hrs

# iterates over each character
# in the command line arguments
from sys import argv

# for every word in the list
for s in argv:
    # for every character in each word in the list
    for c in s:
        print(c, end=" ")
    print()